### ISSUE TITLE
SKS8300-8X

### ISSUE THREAD

**Issue by antt0n** [musashino205/routers#2]
Good morning,

Sorry to bother you.
I saw that you worked several months ago on OpenWRT for the XikeStore SKS8300-8X switch.
I currently have one, but I also see that obviously, OpenWRT is not yet functional on this hardware.
Could you tell me about the progress? Has the porting project been stopped?
I am available to carry out tests, having an SKS8300.

Thanks in advance!

**Comment by musashino205** [comment]
When I worked for SKS8300-8X, the SerDes adapters of SFP+ ports didn't work with the drivers at that time and I suspended the work and started for other tasks (kirkwood, mvebu, etc...).
I'm planning to test with the current (updated) drivers in realtek target when I have enough time.

**Comment by madmalkav** [comment]
Hi, first of all thanks for your work.

ONT-S508CL-8S seems to be the same as SKS8300-8X. Do you have any information about this model? It is possible your work on SKS8300-8X will work on the ONT-S508CL-8S?

**Comment by musashino205** [comment]
I don't have ONT-S508CL-8S and any information about that model, so

> It is possible your work on SKS8300-8X will work on the ONT-S508CL-8S?

It maybe, but I can't say for sure that's the case.

**Comment by madmalkav** [comment]
Thanks for your quick answer!El 18 ene 2025, a las 9:53, INAGAKI Hiroshi ***@***.***> escribió:﻿
I don't have ONT-S508CL-8S and any information about that model, so

It is possible your work on SKS8300-8X will work on the ONT-S508CL-8S?

It maybe, but I can't say for sure that's the case.

**Comment by Curtman** [comment]
> It is possible your work on SKS8300-8X will work on the ONT-S508CL-8S?

I have two of the ONT-S508CL-8S, and I've been testing things out on my spare.  I can confirm that the firmware from the SKS8300-8X seems to run fine on the ONT-S508CL-8S, and OpenWRT dev snapshots function as well as they should.

**Comment by musashino205** [comment]
Yes, that ONTi device was added as a alternative name of SKS8300-8X.

https://github.com/openwrt/openwrt/commit/8b3c845835a02677cc6bf0e225752406a569b8d7

**Comment by udif** [comment]
I suspect that the HICOMDATA HK-S508CL-8S is also identical.

**Comment by ejcabquina** [comment]
Hi all,

Does anyone have any dump of the .bin from its flash chip? (this is using Winbond W25Q256JVFIQ)

**Comment by daniel-dona** [comment]
> Hi all,
> 
> Does anyone have any dump of the .bin from its flash chip? (this is using Winbond W25Q256JVFIQ)

Whole flash contents (from the ONTi switch)

[onti_original.bin.zip](https://github.com/user-attachments/files/23607978/onti_original.bin.zip)

**Comment by netcore-24** [comment]
> Все содержимое флэш-памяти (с коммутатора ONTi)

Could you share the firmware for downloading via the web? I accidentally flashed the original Onti from the same switch, but from a different manufacturer. Everything works, but the default settings don't match the sticker.
